src/vs/editor/contrib/smartSelect/common/smartSelect.ts

```diff
@@ -196,3 +198,84 @@ class ShrinkSelectionAction extends AbstractSmartSelect {
 		});
 	}
 }
+
+@editorAction
+class SelectBracketAction extends EditorAction {
+	constructor() {
+		super({
+			id: 'editor.action.expandSelectionToBrackets',
+			label: nls.localize('smartSelect.expandSelectionToBrackets', "Expand Selection To Brackets"),
+			alias: 'Expand Selection To Brackets',
+			precondition: null,
+			kbOpts: {
+				kbExpr: EditorContextKeys.textFocus,
+				primary: KeyMod.CtrlCmd | KeyMod.Shift | KeyCode.US_SLASH
```

Comment:
Although on further thought this would be a bit of a hassle to press a few times... Maybe something as basic as `Alt + M` would work.